spec/locale.html

```diff
@@ -452,5 +550,265 @@ <h1>
         1. Return the substring of _variants_ from 1.
       </emu-alg>
     </emu-clause>
+
+    <emu-clause id="sec-create-array-from-list-or-restricted" type="abstract operation">
+      <h1>
+        CreateArrayFromListOrRestricted (
+          _list_: a List of ECMAScript language values,
+          _restricted_: an ECMAScript language value*,
+        ): an Array
+      </h1>
+      <dl class="header">
+      </dl>
+      <emu-alg>
+        1. If _restricted_ is not *undefined*, then
+          1. Set _list_ to &laquo; _restricted_ &raquo;.
+        1. Return CreateArrayFromList( _list_ ).
+      </emu-alg>
+    </emu-clause>
+
```

Comment:
This operation isn't carrying its own weight. Please instead inline it at the respective use sites, e.g.
```patch
-        1. Let _restricted_ be _loc_.[[Collation]].
+        1. If _loc_.[[Collation]] is not *undefined*, return « _loc_.[[Collation]] ».
         1. Let _locale_ be _loc_.[[Locale]].
         1. Let _list_ be a List of one or more unique collation types in canonical form (<emu-xref href="#sec-collation-types"></emu-xref>), of those in common use for string comparison in _locale_. The values *"standard"* and *"search"* must be excluded from _list_. The list is sorted according to lexicographic code unit order.
-        1. Return CreateArrayFromListOrRestricted( _list_, _restricted_ ).
+        1. Return CreateArrayFromList(_list_).
```
```suggestion
```